librdfa-rdf4j/README.MD

```diff
@@ -0,0 +1,63 @@
+# Librdfa - RDF4J
+
+RDF4J parser that uses [librdfa](https://github.com/rdfa/librdfa) to parse RDFa to triples.
+
+## Prerequisites
+
+You need to install the [librdfa](https://github.com/rdfa/librdfa) library.
+
+## Install
+
+``` mvn
+<dependency>
+   <groupId>org.apache.any23</groupId>
+   <artifactId>apache-any23-librdfa</artifactId>
+   <version>1.0.0</version>
+</dependency>
+
+<repositories>
+   <repository>
+       <id>librdfa-rdf4j</id>
+       <url>https://raw.github.com/JulioCCBUcuenca/librdfa-java/repository/</url>
+   </repository>
+</repositories>
+```
+
+## Compile
+
+`mvn clean install`
+
+## Use
+
+Add the library and you can parse an `InputStream` as you would do with [`Rio`](http://docs.rdf4j.org/javadoc/2.1/org/eclipse/rdf4j/rio/Rio.html).
+
+``` java
+RDFParser rdfParser = Rio.createParser(RDFFormat.RDFA);
+Model model = new LinkedHashModel();
+rdfParser.setRDFHandler(new StatementCollector(model));
+rdfParser.parse(in, "http://www.example.org./");
+```
+
+## Benchmarking
+
+In general librdfa is 2-5 seconds faster than semargl.  
+
+### librdfa-rdf4j
+- round: 0.11 [+- 0.00] 
```

Comment:
What is the unit of these floating points?

Reply:
Each metric is in milliseconds.